files/riot/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "riotjs",
+  "repo": "muut/riotjs",
+  "files": {
+    "include": ["riot.js", "riot.min.js"]
```

Comment:
I dont see these files here https://github.com/muut/riotjs/tree/v2.0.1
The bot will fail to update them in the future